proxy/context.go

```diff
@@ -233,10 +237,104 @@ func (c *context) clone() *context {
 	return &cc
 }
 
+func (c *context) wasExecuted()bool {
```

Comment:
is there any missing formatting? Wondering if there suppose to be a whitespace before the bool return type.

Reply:
Missign formatting. Done.